src/openrct2/ride/TrackDesign.cpp

```diff
@@ -701,467 +701,506 @@ static void track_design_update_max_min_coordinates(int16_t x, int16_t y, int16_
     gTrackPreviewMax.z = std::max(gTrackPreviewMax.z, z);
 }
 
-/**
- *
- *  rct2: 0x006D0964
- */
-static int32_t track_design_place_scenery(
-    rct_td6_scenery_element* scenery_start, int32_t originX, int32_t originY, int32_t originZ)
+static bool TrackDesignPlaceSceneryTileDrawOutline(LocationXY8 tile)
```

Comment:
Don't use Location8/16/32 for anything new. It's deprecated, because it can be either (big) Coords or TileCoords. Use an explicit struct type.

Reply:
`[[deprecated]]`